libbpf-tools/Makefile

```diff
@@ -111,6 +123,22 @@ $(OUTPUT)/%.bpf.o: %.bpf.c $(LIBBPF_OBJ) $(wildcard %.h) $(ARCH)/vmlinux.h | $(O
 		     -I$(ARCH)/ $(INCLUDES) -c $(filter %.c,$^) -o $@ &&      \
 	$(LLVM_STRIP) -g $@
 
+ifdef ENABLE_MIN_CORE_BTFS
+
+.PHONY: update_btfhub
+update_btfhub: $(BTF_HUB_ARCHIVE)
```

Comment:
might be forgetting some Make quirk, but why extra rule if you can just do that cd + git pull right there in min_core_btf_tar.o rule?